idea/idea-frontend-api/src/org/jetbrains/kotlin/idea/frontend/api/calls/ktCallUtils.kt

```diff
@@ -12,6 +12,11 @@ public fun KtCallTarget.getSuccessCallSymbolOrNull(): KtFunctionLikeSymbol? = wh
     is KtErrorCallTarget -> null
 }
 
+public fun KtCallTarget.getSingleCandidateSymbolOrNull(): KtFunctionLikeSymbol? = when (this) {
```

Comment:
Could you explain the use case for this?

Reply:
It's used here: https://github.com/JetBrains/intellij-kotlin/pull/83/commits/ffa01029428933f27e9b67182d7295d6c88919f3#diff-496d85795f3d93b735bab192eae02991cf2aed3e17cbff9a366b9b08098ecbdfR53

If there are multiple candidates (i.e., ambiguous) we don't really know which candidate to check. We can still add parameter name on a resolve error (`KtErrorCallTarget` branch) as long as there's a single candidate.